htdocs/components/02_LayoutManager.js

```diff
@@ -25,7 +25,7 @@ Ensembl.LayoutManager.extend({
    */
   initialize: function () {
     this.id = 'LayoutManager';
-    
```

Comment:
can you please remove the white space and remove this file from the commit, there is no changes here.

Reply:
You mean add white space?? :)